app/api/ada/lib/storage/database/genericBip44/tables.js

```diff
@@ -0,0 +1,391 @@
+// @flow
+
+import {
+  ConceptualWalletSchema,
+  KeySchema,
+} from '../uncategorized/tables';
+import { Type } from 'lovefield';
+import type { lf$schema$Builder } from 'lovefield';
+
+export type Bip44DerivationInsert = {|
+  PublicKeyId: number | null,
+  PrivateKeyId: number | null,
+  IsHardened: boolean,
+|};
+export type Bip44DerivationRow = {|
+  Bip44DerivationId: number, // serial
+  ...Bip44DerivationInsert,
+|};
+export const Bip44DerivationSchema: {
+  name: string,
+  properties: $ObjMapi<Bip44DerivationRow, ToSchemaProp>
+} = {
+  name: 'Bip44Derivation',
+  properties: {
+    Bip44DerivationId: 'Bip44DerivationId',
+    IsHardened: 'IsHardened',
+    PrivateKeyId: 'PrivateKeyId',
+    PublicKeyId: 'PublicKeyId',
+  }
+};
+
+export type Bip44WrapperInsert = {|
+  ConceptualWalletId: number,
+  IsBundled: boolean,
+  SignerLevel: number | null,
+  PublicDeriverLevel: number,
+  PrivateDeriver: number | null,
```

Comment:
I didn't find `PrivateDeriver` on the diagram nor the spec. Is it a typo or intended to represent something new?

Reply:
https://github.com/Emurgo/yoroi-frontend/pull/701#discussion_r300935851